spring-batch-bigquery/pom.xml

```diff
@@ -34,70 +47,40 @@
         </developer>
     </developers>
 
-    <licenses>
-        <license>
-            <name>Apache-2.0</name>
-            <url>https://www.apache.org/licenses/LICENSE-2.0.txt</url>
-            <distribution>repo</distribution>
-        </license>
-    </licenses>
-
     <scm>
         <url>https://github.com/spring-projects/spring-batch-extensions</url>
         <connection>git://github.com/spring-projects/spring-batch-extensions.git</connection>
         <developerConnection>git@github.com:spring-projects/spring-batch-extensions.git</developerConnection>
     </scm>
 
-    <properties>
-        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-
-        <!-- Dependent on Spring Batch core -->
-        <java.version>17</java.version>
-        <logback.version>1.5.13</logback.version>
-    </properties>
-
     <dependencies>
+        <!-- Compile -->
         <dependency>
-            <groupId>org.springframework.batch</groupId>
-            <artifactId>spring-batch-core</artifactId>
-            <version>5.1.2</version>
+            <groupId>com.fasterxml.jackson.dataformat</groupId>
+            <artifactId>jackson-dataformat-csv</artifactId>
         </dependency>
-
         <dependency>
             <groupId>com.google.cloud</groupId>
             <artifactId>google-cloud-bigquery</artifactId>
-            <version>2.42.0</version>
+            <version>2.45.0</version>
         </dependency>
-        <dependency>
-            <groupId>com.fasterxml.jackson.dataformat</groupId>
-            <artifactId>jackson-dataformat-csv</artifactId>
-            <version>2.17.2</version>
-        </dependency>
-
         <dependency>
             <groupId>org.apache.commons</groupId>
             <artifactId>commons-lang3</artifactId>
-            <version>3.16.0</version>
         </dependency>
-
-
-        <!-- Test Dependencies -->
+        <dependency>
+            <groupId>org.springframework.batch</groupId>
+            <artifactId>spring-batch-core</artifactId>
+        </dependency>
+        <!-- Test -->
```

Comment:
One empty line before test dependencies to visually distinguish 2 groups of dependencies.